### PR TITLE
Feature/besluiten api plugin plugin definition

### PR DESCRIPTION
**Pull request by marijnritense**

### CODE REVIEW

File: zgw/besluiten-api/build.gradle [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    testImplementation "org.junit.jupiter:junit-jupiter"
+    //Kotlin
+    testImplementation "org.mockito.kotlin:mockito-kotlin:${mockitoKotlinVersion}"
+    testImplementation "com.fasterxml.jackson.module:jackson-module-kotlin"
+
+    jar {
+        enabled = true
+        manifest {
+            attributes("Implementation-Title": "ZGW Zaken API plugin spring boot starter")
```
Comment (luis-soares-ritense) [inline zgw/besluiten-api/build.gradle:39]: Shouldn't it be Besluiten API?

Comment (marijnritense) [reply zgw/besluiten-api/build.gradle]: fixed

File: zgw/besluiten-api/src/main/kotlin/com/ritense/besluitenapi/BesluitenApiPlugin.kt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import com.ritense.plugin.annotation.PluginProperty
+import com.ritense.zgw.Rsin
+import mu.KLogger
+import mu.KotlinLogging
+import java.net.URI
+
+@Plugin(key = BesluitenApiPlugin.PLUGIN_KEY,
+    title = "Besluiten API",
+    description = "Connects to the Besluiten API")
+class BesluitenApiPlugin {
```
Comment (luis-soares-ritense) [inline zgw/besluiten-api/src/main/kotlin/com/ritense/besluitenapi/BesluitenApiPlugin.kt:29]: missing configuration name property 

The new 'Besluiten API' plugin can be configured with the properties:
Configuration name
URL (String)

RSIN (String)

Authentication plugin configuration

**Review by luis-soares-ritense — APPROVED** [review]

File: settings.gradle [hunk 3]
```diff
@@ -77,5 +77,6 @@ include(
     "zgw:zaken-api",
     "zgw:portaaltaak"
 )
-
+include 'zgw:besluiten-api'
+findProject(':zgw:besluiten-api')?.name = 'besluiten-api'
```
Comment (Klaas-Ritense) [inline settings.gradle:81]: Not sure why this is needed. Can't it be in the include list above?

Also, can the new besluiten-api dependency be added to the /app/gzac/build.gradle?

Comment (marijnritense) [reply settings.gradle]: Missed that, guess it was generated. Fixed it, and also added the besluiten-api dependency to the gzac module

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**